translate/storage/test_txt.py

```diff
@@ -33,7 +33,13 @@ def test_multipleblocks(self):
         txtfile = self.txtparse(txtsource)
         assert len(txtfile.units) == 3
         print(txtsource)
-        print(txtfile.serialize())
+        print(bytes(txtfile))
         print("*%s*" % txtfile.units[0])
-        assert txtfile.serialize().decode('utf-8') == txtsource
+        assert bytes(txtfile).decode('utf-8') == txtsource
         assert self.txtregen(txtsource) == txtsource
+
+    def test_getoutput(self):
+        # Test deprecated method still works
```

Comment:
pytest allows to ensure that this is deprecated. See for an example https://github.com/translate/translate/blob/b512b51d64ab3e3c1a022c5541ac29b0abb91e9a/translate/storage/test_properties.py#L52

Related to that example you might want to drop the now unused `deprecated_call` import in that file.